centaurCwlRunner/src/main/scala/centaur/cwl/Outputs.scala

```diff
@@ -68,6 +68,13 @@ object Outputs {
 
   //In an Ideal world I'd return a Coproduct of these types and leave the asJson-ing to the handleOutput
   def resolveOutput(jsValue: JsValue, mot: MyriadOutputType): Json  = {
+    mot match {
+      case Inl(moit) => resolveOutputViaInnerType(jsValue, moit)
+      case Inr(Inl(arrayMoit)) => resolveOutputViaInnerType(jsValue, arrayMoit.head)
+      case _ => ??? //Impossible to reach this code
```

Comment:
I'm not advocating for polys everywhere but it in this case it would avoid this no ?

Reply:
Yeah... I'll fix it up.

Reply:
I keep forgetting the `unapply` approach!